vhdbuilder/packer/packer_source.sh

```diff
@@ -186,10 +186,13 @@ copyPackerFiles() {
 
   NOTICE_SRC=/home/packer/NOTICE.txt
   NOTICE_DEST=/NOTICE.txt
-  if [[ ${UBUNTU_RELEASE} == "16.04" ]]; then
-    SSHD_CONFIG_SRC=/home/packer/sshd_config_1604
-  elif [[ ${UBUNTU_RELEASE} == "18.04" && ${ENABLE_FIPS,,} == "true" ]]; then
-    SSHD_CONFIG_SRC=/home/packer/sshd_config_1804_fips
+
+  if [[ $OS != $MARINER_OS_NAME ]]; then
```

Comment:
is this condition actually necessary?

Reply:
No, but I do feel that it makes it more clear. Perhaps flipping the if would add even more clarity -- if ubuntu look at these things.

Reply:
No. I'll remove.